rotkehlchen/externalapis/defillama.py

```diff
@@ -101,7 +101,15 @@ def _get_asset_id(self, asset: AssetWithOracles) -> str:
             asset = asset.resolve_to_evm_token()
             # The evm chain names we have so far match perfectly except for 'polygon pos'
             # which is expected as polygon by the defillama api
-            chain_name = 'polygon' if asset.chain_id == ChainID.POLYGON_POS else str(asset.chain_id)  # noqa: E501
+            if asset.chain_id == ChainID.POLYGON_POS:
```

Comment:
have you checked all other chains we have? The comment needs adjusting above ^

Reply:
Updated. Yes in the test